main.tf

```diff
@@ -69,26 +65,13 @@ resource "google_project_iam_member" "sa_logging_log_writer" {
   member  = "serviceAccount:${google_service_account.sa.email}"
 }
 
+//TODO do we need this? Or instead should we pass service account key as variable?
```

Comment:
Yes but if you want to have an optional override for testing scenarios go for it!